specs/7.0/panels/_thresholds.yml

```diff
@@ -0,0 +1,24 @@
+thresholds:
+  type: array
+  items:
+    type: object
+    properties:
+      colorMode:
+        title: Threshold Color
+        type: string
+        default: critical
+      fill:
+        title: Fill
+        type: boolean
+        default: true
+      line:
+        title: Line
+        type: boolean
+        default: true
+      op:
+        title: Operator
+        type: string
+        default: gt
+      value:
+        title: value
+        type: number 
```

Comment:
The filename update makes sense, but this new structure seems wrong. A thresholds object should look like this:

```
"thresholds": {
  "mode": "absolute",
  "steps": [
    {
      "color": "yellow",
      "value": null
    },
    {
      "color": "green",
      "value": 0.7
    },
    {
      "color": "red",
      "value": 0.9
    }
  ]
}
```

The deleted `_threshold.yml` file captures this schema.

Reply:
Sorry for the drive by comment, but I came across this PR whilst working on an experiment to port some of our Grafana JSON snapshots to Grafonnet 7.0. In one of our graphs we have a threshold like this:

```json
{
  "$": "object:458",
  "colorMode": "critical",
  "fill": true,
  "line": true,
  "op": "lt",
  "value": 1,
  "yaxis": "left"
}
```

From looking at the graph with and without this threshold, it seems that in this form it will draw a line on the graph for the threshold. The current schema (the deleted `_threshold.yml`) does not allow for `colorMode`, `fill`, `line`, `op`, etc. to be specified, whereas the one in this PR does.